examples/vision/text_image_denoiser.py

```diff
@@ -0,0 +1,295 @@
+"""
+Title: Text Image Denoiser.
+Author: [Anish B](https://twitter.com/anishhacko)
+Date created: 2021/01/18
+Last modified: 2022/01/27
+Description: Example of Text Image Denoiser coupled with Tesseract OCR engine to improve text readability.
+"""
+"""
+## Introduction
+
+This Example explains a simple **Text Image Denoiser for OCR** using U-Net based
+Architectutre. **Autoencoders** are mostly employed for **Image Restoration** problems
+but in recent times **U-Net** based architectures with **skip-connections** have gained
+popularity for several Image-to-Image tasks. We aim to solve the problem with simple
+**Pretrained U-Net as Encoder** and **Efficient Sub-Pixel CNN as Decoder**. The problem
```

Comment:
We aim to solve the problem with simple pretrained **U-Net** as Encoder and **Efficient Sub-Pixel** CNN as Decoder